<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DlgEditSize</class>
 <widget class="QDialog" name="DlgEditSize">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>319</width>
    <height>159</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Edit coordinates</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_6">
   <item row="0" column="0" rowspan="4">
    <widget class="QGroupBox" name="groupboxCoords">
     <property name="title">
      <string>Coordinates</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_5">
      <item row="0" column="0">
       <widget class="QWidget" name="widgetX" native="true">
        <layout class="QGridLayout" name="gridLayout">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item row="0" column="0">
          <widget class="QLabel" name="labelX">
           <property name="text">
            <string>X:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QSpinBox" name="spinboxX">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="maximum">
            <number>16777215</number>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QWidget" name="widgetY" native="true">
        <layout class="QGridLayout" name="gridLayout_2">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item row="0" column="0">
          <widget class="QLabel" name="labelY">
           <property name="text">
            <string>Y:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QSpinBox" name="spinboxY">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="maximum">
            <number>16777215</number>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QWidget" name="widgetW" native="true">
        <layout class="QGridLayout" name="gridLayout_3">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item row="0" column="0">
          <widget class="QLabel" name="labelW">
           <property name="text">
            <string>Width:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QSpinBox" name="spinboxW">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimum">
            <number>0</number>
           </property>
           <property name="maximum">
            <number>16777215</number>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QWidget" name="widgetH" native="true">
        <layout class="QGridLayout" name="gridLayout_4">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item row="0" column="0">
          <widget class="QLabel" name="labelH">
           <property name="text">
            <string>Height:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QSpinBox" name="spinboxH">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="maximum">
            <number>16777215</number>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QPushButton" name="btnApply">
     <property name="text">
      <string>Adjust</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QPushButton" name="btnReset">
     <property name="text">
      <string>Reset</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QPushButton" name="btnExit">
     <property name="text">
      <string>Close</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btnApply</sender>
   <signal>clicked()</signal>
   <receiver>DlgEditSize</receiver>
   <slot>slotApply()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>272</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>159</x>
     <y>67</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnExit</sender>
   <signal>clicked()</signal>
   <receiver>DlgEditSize</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>272</x>
     <y>49</y>
    </hint>
    <hint type="destinationlabel">
     <x>159</x>
     <y>67</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnReset</sender>
   <signal>clicked()</signal>
   <receiver>DlgEditSize</receiver>
   <slot>slotReset()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>272</x>
     <y>114</y>
    </hint>
    <hint type="destinationlabel">
     <x>159</x>
     <y>67</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>slotApply()</slot>
  <slot>slotReset()</slot>
 </slots>
</ui>
